meteor-app/imports/ui/pages/workspace-charts/style.less

```diff
@@ -9,23 +9,61 @@
     position: fixed;
     background-color: white;
     top: 0;
-    width: 100%;
     left: 0;
     right: 0;
     padding: 10px;
     z-index: 10;
-    min-width: 300px;
-    min-height: 60px;
+    height: 30px;
+    
+    display: -ms-flexbox;
```

Comment:
Use pre-defined style components (e.g. `flex-row`). Also you don't need to include cross-platform prefixes anymore as an auto-prefixer has been incorporated.